FlaxEditor/Windows/GameWindow.cs

```diff
@@ -287,6 +289,18 @@ public override void Draw()
                     Render2D.DrawRectangle(new Rectangle(new Vector2(4), Size - 8), Color.Orange * alpha);
                 }
             }
+
+            var selection = Editor.Instance.SceneEditing.Selection;
+                
+            selection.ForEach((node) =>
+            {
+                if (node.EditableObject is UIControl controlActor)
+                {
+                    if (controlActor.Control != null)
+                        Render2D.DrawRectangle(controlActor.Control.Bounds, Color.Green, 5f);
```

Comment:
I suppose the 5f could be reduced to 1f to be more consistent with the existing selections in Flax

Reply:
It's nearly invisible with 1F.

Reply:
Ok, that makes sense. Does 2f look fine?

By the way, if you want to use the user set selection outline color, you should be able to access it with `Editor.Options.Visual.SelectionOutlineColor0`